tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
```suggestion
    "module": "ESNext",
```

Reply:
This may cause issues with Gatsby, so it's probably better to use commonjs like you had it